pkg/swap/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM alpine:3.5
+
+# swap command - only minimal Alpine install
+
+# use util-linux to get swapfile utils
+#   at some point, hopefully use our own mkswap and swapon
+RUN apk add --update util-linux
+
+# add the entrypoint and make it executable
+COPY . ./
+RUN chmod +x swap.sh
```

Comment:
I just like the uniformity that all Dockerfiles should be 
```
FROM x
COPY . ./
CMD ...
```
and rarely anything else where possible - the less processing of the inputs the better...